statsform.module

```diff
@@ -387,4 +387,23 @@ function statsform_form_page_two_submit($form, &$form_state) {
   // The first element in the array is the path to redirect to, and the second
   // element in the array is the array of options. For more information on the
   // available options, see http://api.drupal.org/url.
-}
\ No newline at end of file
+}
+
+/**
+ * Implements hook_page_alter().
+ */
+function statsform_page_alter(&$page) {
+  $page['#theme'] = 'hijack';
```

Comment:
here is where we are telling the page to use a specific theme function that happens to be a the "page" template level.

We will have to add some logic so that the page in not hijacked on every request, just the path we want.